src/styles.scss

```
@@ -1,356 +1,357 @@
-    cursor: default;
-    &:hover{
-        background-color: $mainPurple;
-    }
+button:disabled {
+  opacity: 0.5;
+  cursor: default;
+  &:hover {
+    background-color: $mainPurple;
+  }
 }
 // FORM CHECKBOX //
```

Stylelint (17.12.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)